src/qlever/commands/example_queries.py

```diff
@@ -603,3 +785,125 @@ def execute(self, args) -> bool:
 
         # Return success (has nothing to do with how many queries failed).
         return True
+
+    def get_record_for_yaml(
+        self,
+        query: str,
+        sparql: str,
+        client_time: float,
+        result: str | dict[str, str],
+        result_size: int | None,
+        accept_header: str,
+    ) -> dict[str, Any]:
+        """
+        Construct a dictionary with query information for yaml file
```

Comment:
This function should have a better name and comment. In particular, the input file can be a YML as well, so `yaml` is not a unique reference